app/assets/javascripts/backbone/views/error_handler.coffee

```diff
@@ -0,0 +1,25 @@
+class MS.Views.ErrorHandler extends Backbone.View
+
+  el: 'body'
+
+  initialize: ->
+    MS.loadingIndicator = @$('#glb-load-ind')
```

Comment:
This doesn't belong in this view, I think. It's not strictly related to error handling.